docs/mapping/freiburg.md

```diff
@@ -0,0 +1,74 @@
+|----------------------------|--------------------------|-------------|---------------------------------|---------------------------------------------------------------------|
+| ogc_fid                    | integer                  | 1           | uid                             |                                                                     |
+| kapazitaet                 | integer                  | 1           | capacity                        |                                                                     |
+| name                       | string                   | 1           | name                            |                                                                     |
+| nummer                     | string                   | ?           | name                            |                                                                     |
+| kategorie                  | string                   | 1           | #ParkingSiteType                |                                                                     |
+
+
+## ParkingSites for P+R Realtime and Static
+
+Attributes which are set statically:
+* `type` is always set to `OFF_STREET_PARKING_GROUND`
+* `park_and_ride_type` is always set to `['YES']`
+* `purpose` is always set to `CAR`
+* `has_realtime_data` is always set to `True`
+
+A `ParkingSites` provides static and realtime data for a Park and Ride `ParkingSite`.
+
+| Field                      | Type                     | Cardinality | Mapping                         | Comment                                                             |
+|----------------------------|--------------------------|-------------|---------------------------------|---------------------------------------------------------------------|
+| park_id                    | integer                  | 1           | uid                             |                                                                     |
+| name                       | string                   | 1           | name                            |                                                                     |
+| obs_max                    | integer                  | ?           | realtime_capacity/capacity      |                                                                     |
+| obs_free                   | integer                  | ?           | realtime_free_capacity          |                                                                     |
+| obs_ts                     | datetime                 | 1           | realtime_data_updated_at        |                                                                     |
+| obs_state                  | integer                  | ?           | #RealtimeOpeningStatus          |                                                                     |
```

Comment:
Please add a link, and make an `RealtimeOpeningStatus` type based on that. The mapping is `realtime_opening_status`.

Also: The cardinality in your validator is 1, not ?. What is the correct value?